src/config_test.rs

```diff
@@ -268,4 +268,31 @@ mod get_hook_script_tests {
         let result = get_hook_script(contents, "pre-commit");
         assert_eq!(result.unwrap(), "cargo test");
     }
+
+    #[test]
+    fn returns_result_when_value_array() {
+        let contents = "[hooks]
+            pre-commit = [
+                'cargo test',
+                'cargo fmt'
+            ]
+        ";
```

Comment:
Don't feel obligated to change this, but we could change these to use raw string literals instead. We haven't done this in any of the other tests though so this is fine, mostly sharing for awareness.

```suggestion
        let contents = r#"[hooks]
            pre-commit = [
                "cargo test",
                "cargo fmt"
            ]
        "#;
```

Reply:
Also added this suggestion. I can probably do a PR to change this for the rest of the tests sometime later this week.